docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3.7'
+services:
+  mqtt-broker_2:
+    image: eclipse-mosquitto:2.0.18
+    container_name: mqtt-broker_2
+    ports:
+      - "1885:1883"
+    volumes:
+      - ./mosquitto-custom/mosquitto.conf:/mosquitto/config/mosquitto.conf
+    restart: unless-stopped
+
+  grafana:
+    image: grafana/grafana:latest
+    container_name: my_grafana_1
+    environment:
+      - GF_INSTALL_PLUGINS=grafana-mqtt-datasource
+    volumes:
+      - ./grafana-custom/provisioning/dashboards/dashboards.yaml:/etc/grafana/provisioning/dashboards/dashboards.yaml
+      - ./grafana-custom/provisioning/datasource/mqtt-datasource.yaml:/etc/grafana/provisioning/datasources/mqtt-datasource.yaml
+      - ./grafana-custom/dashboards/:/var/lib/grafana/dashboards
+    ports:
+      - "3001:3000"
+    depends_on:
+      - mqtt-broker_2
+
+  mqtt-runner-1:
```

Comment:
can you give these containers more meaningful names instead of `mqtt-runner` - ie this one could be fps-publisher-1 or something